File: app/src/androidTest/java/com/github/fo2rist/mclaren/pages/PreviewPage.java
package com.github.fo2rist.mclaren.pages;

import android.support.test.espresso.ViewInteraction;

import com.github.fo2rist.mclaren.R;

import static android.support.test.espresso.Espresso.onView;
import static android.support.test.espresso.matcher.ViewMatchers.withId;

public class PreviewPage {
    public ViewInteraction onToolbarImage() {
        return onView(withId(R.id.image_header));
    }

    public ViewInteraction onWebView() {
        return onView(withId(R.id.web_view));
    }

    public ViewInteraction onGalleyView() {
        return onView(withId(R.id.images_pager));
    }
}

File: app/src/androidTest/java/com/github/fo2rist/mclaren/pages/PreviewPage.kt
package com.github.fo2rist.mclaren.pages

import com.agoda.kakao.pager.KViewPager
import com.agoda.kakao.screen.Screen
import com.agoda.kakao.web.KWebView
import com.github.fo2rist.mclaren.R

class PreviewPage : Screen<PreviewPage>() {

    val toolbarImages = KViewPager { withId(R.id.header_image_pager) }

    val webView = KWebView { withId(R.id.web_view) }

    val galleryView = KViewPager { withId(R.id.images_pager) }
}
